interfaces/builtin/block_devices.go

```diff
@@ -53,14 +53,32 @@ const blockDevicesConnectedPlugAppArmor = `
 /sys/devices/**/block/** r,
```

Comment:
Up above the blockDevicesSummary, can you adjust this:
```
// Only allow raw disk devices; not loop, ram, CDROM, generic SCSI, network,
// tape, raid, etc devices or disk partitions
```
to be:
```
// Only allow raw disk devices; not loop, ram, CDROM, generic SCSI, network,
// tape, raid, etc devices or disk partitions. For some devices, allow controller
// character devices since they are used to configure the corresponding block
// device.
```